helpers/middleman/blog.rb

```diff
@@ -3,22 +3,50 @@
 
 module MiddlemanBlogHelpers
 
-  def blog_posts(order: nil)
+  def blog_posts(order: :latest_first)
     @blog_posts ||= {}
     return @blog_posts[order] if @blog_posts[order]
 
     @blog_posts[order] ||= case order
-      when :youngest_first
+      when :oldest_first
         data.blog.posts.values.sort do |a,b|
-          a['created_at'] <=> b['created_at']
+          a['created_at'].to_i <=> b['created_at'].to_i
         end
-      when :oldest_first
+      when :latest_first
         data.blog.posts.values.sort do |a,b|
-          b['created_at'] <=> a['created_at']
+          b['created_at'].to_i <=> a['created_at'].to_i
         end
       else
-        data.blog.posts
+        data.blog.posts.values
+      end
+  end
+
+  def blog_posts_for_menu
```

Comment:
@irwalker I'm going to call you my Ruby reviewer.  I added this and a whole slew of test coverage…plus https://github.com/sharesight/www.sharesight.com/pull/253 (sorry to jam all of that in here).

1. Had to filter the blog posts based on categories…was a bit more overkill than I thought, but this was the result.
2. Decided to fix the order of the blog posts (they were named the opposite of their function).
3. Allowed me to test all of this.